PrReviewCode/Program.cs

```diff
@@ -48,4 +50,6 @@ public static void Main(string[] args)
 
         app.Run();
     }
-}
\ No newline at end of file
+    
```

Comment:
Uncaught exception in `/weatherforecast` endpoint causes HTTP 500 errors and exposes stack traces. This breaks API stability and security.

 **SUGGESTION:**

```suggestion
 // Handle or remove this exception
```